lib/sky-pages-module-generator.js

```diff
@@ -14,6 +17,32 @@ const routeGenerator = require('./sky-pages-route-generator');
  * @returns {string} source
  */
 function getSource(skyAppConfig) {
+  const cssFilePaths = get(skyAppConfig, 'skyux.app.styles', []);
+
+  let styles = '';
+  const numFiles = cssFilePaths.length;
```

Comment:
This is non-blocking, but I would've expected this const to be up against the `cssFilePaths` const, then followed by the `let`.  Just stood out a bit.